packages/domain/src/use-cases/profile/UpdateFollowPolicy.ts

```diff
@@ -24,6 +25,12 @@ export type NoFeeFollowPolicy = {
   type: FollowPolicyType.ANYONE | FollowPolicyType.ONLY_PROFILE_OWNERS | FollowPolicyType.NO_ONE;
 };
 
+export type UnsupportedFollowPolicy = {
+  type: FollowPolicyType.UNKNOWN;
+};
+
+export type FollowPolicy = ChargeFollowPolicy | NoFeeFollowPolicy | UnsupportedFollowPolicy;
```

Comment:
ah ok, this is the source of confusion regarding CA we were discussing in Slack.

I think this:
```ts
export type UnsupportedFollowPolicy = {
  type: FollowPolicyType.UNKNOWN;
};

export type FollowPolicy = ChargeFollowPolicy | NoFeeFollowPolicy | UnsupportedFollowPolicy;
```

is better suited to live in `packages/api-bindings/src/graphql/FollowPolicy.ts`.

Reply:
Discussed on a call and changes implemented - much more streamlined approach 👍